roles/ceph-osd/tasks/scenarios/collocated.yml

```diff
@@ -57,4 +69,14 @@
     - "{{ devices }}"
   when:
     - not containerized_deployment
-    - item.0.partitions|length == 0
\ No newline at end of file
+  register: osd
+
+- set_fact: osd_created=True
+  when: osd.changed
+
+- fail:
+    msg: >
+      OSD was never created.  If using not using osd_auto_discovery, ensure the
```

Comment:
extra space between `.` and `If`.

Reply:
I usually use 2 spaces after periods.  I'll change them all to 1.